web/src/admin/common/ak-flow-search/FlowSearch.ts

```diff
@@ -46,8 +46,8 @@ export class FlowSearch<T extends Flow> extends CustomListenerElement(AKElement)
      *
      * @attr
      */
-    @property({ attribute: false })
```

Comment:
This was the misunderstanding; this is old syntax for using `@property` as `@state`, and it blocked the correct interpretation of inbound strings.  It does allow `currentFlow` to be either `currentFlow` or `.currentFlow` (note leading period) without breaking anything.